doc/examples/numpy_operations/plot_structuring_elements.py

```diff
@@ -0,0 +1,58 @@
+"""
+=============================
+Generate structuring elements
+=============================
+
+This example shows how to use functions in :py:module:`skimage.morphology`
+to generate structuring elements.
+The title of each plot indicates the call of the function.
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+
+from skimage.morphology import square, rectangle, diamond, disk, cube, \
+    octahedron, ball, octagon, star
+
+# Generate 2D and 3D structuring elements.
+# [(title, data)]
+struc_2d = [
+    ("square(5)", square(5)),
+    ("rectangle(5, 3)", rectangle(5, 3)),
+    ("diamond(5)", diamond(5)),
+    ("disk(7)", disk(7)),
+    ("octagon(4, 7)", octagon(4, 7)),
+    ("star(6)", star(6))
+]
```

Comment:
```suggestion
struc_2d = {
    "square(5)": square(5),
    "rectangle(5, 3)": rectangle(5, 3),
    "diamond(5)": diamond(5),
    "disk(7)": disk(7),
    "octagon(4, 7)": octagon(4, 7),
    "star(6)": star(6)
}
```

This is optional, but you may then use then `for title, struc in struc_2D.item()` in the following loop.